contract/src/lib.rs

```diff
@@ -93,12 +91,20 @@ pub struct BtcLightClient {
 
     // GC threshold - how many blocks we would like to store in memory, and GC the older ones
     gc_threshold: u64,
+
+    // Expected time bitween blocks adjustment in seconds,
+    // for bitcoin = 600 * 2016, litecoin = 150 * 2016, dogecoin = 60 * 1
+    expected_time_secs: u64,
```

Comment:
Let's hardcode these values instead of passing it and storing it in the storage.
example:
https://github.com/Near-One/rainbow-bridge/blob/b54f0c92f6e3507abd3a0acc0e55a4aa0de0611f/contracts/near/eth2-utility/src/consensus.rs#L60

Reply:
0182deeed6891305ce3cdbd4cd7a85455fa80eee